README.md

```diff
@@ -1,44 +1,89 @@
-
-  - Base
-  - React (extends the `base`)
-  - React Native (extends `react`, `base`)
+This repository includes a collection of eslint configs that can be combined to lint just about any project, and a default config for all projects that follow our recent patterns.
 
 
 ## Install
 
-Install a specific version of the eslint-config in your package.json. 
+Install a specific version of the eslint-config with NPM. You can see a full list of versions [here](https://github.com/dabapps/eslint-config/releases).
+
+```shell
+npm i https://github.com/dabapps/eslint-config.git#v3.0.0 --save-dev
+```
+
+This will update your package.json automatically.
 
 ```json
 "devDependencies": {
-  "eslint-config": "dabapps/eslint-config#2.0.0",
+  "eslint-config": "https://github.com/dabapps/eslint-config.git#v3.0.0",
 }
 ```
 
-Then you have to choose between the different configs, depending on the project type. Currently there are 3 different configs.
 
-  - base
-  - react
-  - react-native
+## Configuration
 
 
-## Running eslint
+### NPM scripts
 
-Add an `.eslintrc` file in the root of your project that extends a particular config.
+Add the following scripts to your package.json
+
+```json
+{
+  "scripts": {
+    "lint": "npm run lint-src && npm run lint-tests",
+    "lint-src": "eslint -c .eslintrc.json src/",
+    "lint-tests": "eslint -c .eslintrc-test.json tests/"
+  }
+}
+```
+
+
+### Default config
+
+Create an `.eslintrc.json` in the route of the project. For most projects you can use the default config like so:
+
+```json
+{
+  "extends": [
+    "eslint-config"
+  ]
+}
+```
 
-    {
-      "extends": "eslint-config-dabapps/base/.eslintrc", // For Base
-      "extends": "eslint-config-dabapps/react/.eslintrc", // For React
-      "extends": "eslint-config-dabapps/react-native/.eslintrc", // For React-Native
-    }
+Explanation of the default config below
 
-It's recommeneded to then run this in your test script e.g.
 
-    "scripts": {
-      "eslint": "eslint 'src/js/'",
-      "test": "npm run eslint && npm run jest",
-    }
+### Custom config
+
+But if you want to customise the project because, for example, this project is using a different ECMA version, module system, or framework, you can use a combination of the other configs available in this module. To replicate the default config for example, you could use the following:
+
+```json
+{
+  "extends": [
+    "eslint-config/commonjs",
+    "eslint-config/browser",
+    "eslint-config/es6",
+    "eslint-config/react"
+  ]
+}
+```
+
+There's no need to extend the base config as all other configs extend the base anyway.
+
+
+### Test config
+
+If your project has tests (if it doesn't you're fired), create a second config named `.eslintrc-test.json`. This way you can have custom linting for your tests. A common pattern is to extend our existing `.eslintrc.json` and one of our test configs to allow globals like `describe` and `it` e.g.
```

Comment:
As discussed this will break our editor plugins - Just one `.eslintrc` file per project should be enough?

![https://media.giphy.com/media/O9BPkYr89lK2A/giphy.gif](https://media.giphy.com/media/O9BPkYr89lK2A/giphy.gif)